apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/BigQueryRowTest.cs

```diff
@@ -68,6 +70,7 @@ public void Scalars()
             Assert.Equal(new DateTime(2017, 8, 9, 12, 34, 56, 123, DateTimeKind.Utc), (DateTime)row["dateTime"]);
             Assert.Equal(new TimeSpan(0, 12, 34, 56, 123), (TimeSpan)row["time"]);
             Assert.Equal(BigQueryNumeric.Parse("1234567890123456789012345678.123456789"), (BigQueryNumeric) row["numeric"]);
+            Assert.Equal(BigQueryGeography.Parse("POINT(1 2)"), (BigQueryGeography) row["geography"]);
```

Comment:
super-nit: There's inconsistency on whether to have a space after a cast.
(Doesn't need fixing now) (or possibly at all)

Reply:
I think this is because I've fixed my settings (to include the space). Won't bother changing the rest now.